### PR TITLE
docs(orc8r): Adds docs for Charmed Magma 1.8

### PR DESCRIPTION
**Pull request by Gmerold**
## Summary

- Adds release docs for Charmed Magma 1.8
- Adds a complete tutorial for creating a private LTE network using Charmed Magma

## Test Plan

N/A

## Additional Information

- [ ] This change is backwards-breaking

## Security Considerations

N/A

### CODE REVIEW

**Comment by Gmerold** [comment]
@maxhbr The CI failures don't seem to be related to my changes. Would you mind trying to re-run the failing checks please? Thanks!

File: docs/docusaurus/versioned_docs/version-1.8.0/orc8r/deploy_using_juju.md [hunk 1]
```diff
@@ -7,11 +7,104 @@ original_id: deploy_using_juju
...
+Magma Orchestrator and configure DNS A records. For more information on Charmed Magma, please visit
+the project's [homepage](https://canonical.github.io/charmed-magma/1.8/).
 
-To use officially released Charmed Magma 1.6, please refer to the
-[official Charmed Magma 1.6 docs](https://canonical.github.io/charmed-magma/1.6/).
+## Pre-requisites
 
-To try out the latest version of Charmed Magma please follow instructions
-from the [official docs](https://canonical.github.io/charmed-magma/main/).
+- Ubuntu 20.04 machine with internet access
```
Comment (gruyaume) [inline docs/docusaurus/versioned_docs/version-1.8.0/orc8r/deploy_using_juju.md]: Any machine that can install Juju and kubectl would do the job

File: docs/docusaurus/versioned_docs/version-1.8.0/orc8r/deploy_using_juju.md [hunk 2]
```diff
@@ -7,11 +7,104 @@ original_id: deploy_using_juju
...
+- A public domain
+
+## Set up your management environment
+
+From a Ubuntu 20.04 machine, install the following tools:
```
Comment (gruyaume) [inline docs/docusaurus/versioned_docs/version-1.8.0/orc8r/deploy_using_juju.md]:
```suggestion
From your machine, install the following tools:
```

File: docs/docusaurus/versioned_docs/version-1.8.0/orc8r/deploy_using_juju.md [hunk 3]
```diff
@@ -7,11 +7,104 @@ original_id: deploy_using_juju
...
+a Juju controller on it.
+
+1. [MicroK8s](https://juju.is/docs/olm/microk8s)
+2. [Google Cloud (GKE)](https://juju.is/docs/olm/google-kubernetes-engine-(gke))
+3. [Amazon Web Services (EKS)](https://juju.is/docs/olm/amazon-elastic-kubernetes-service-(amazon-eks)#heading--install-the-juju-client)
+4. [Microsoft Azure (AKS)](<https://juju.is/docs/olm/azure-kubernetes-service-(azure-aks)>)
+
+## Deploy charmed Magma Orchestrator
+
+From your Ubuntu machine, create an `overlay.yaml` file that contains the following content:
```
Comment (gruyaume) [inline docs/docusaurus/versioned_docs/version-1.8.0/orc8r/deploy_using_juju.md]:
```suggestion
From your machine, create an `overlay.yaml` file that contains the following content:
```

File: docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+## Requirements
+
+### Management machine
+
+- Ubuntu machine with internet access
+
+### Access Gateway machine
+
+The Access Gateway must be installed on an Ubuntu machine with the following specifications:
```
Comment (gruyaume) [inline docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md]:
```suggestion
The Access Gateway must be installed on an machine with the following specifications:
```

File: docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+    Installing this charm will affect the target computer's networking configuration. Make sure it is installed on designated hardware (personal computers are strongly discouraged).
+
+!!! info
+
+    Some clouds like AWS use newer kernel versions by default. If you want to downgrade your kernel, please refer to the following [guide](https://discourse.ubuntu.com/t/how-to-downgrade-the-kernel-on-ubuntu-20-04-to-the-5-4-lts-version/26459).
+
+!!! info
+
+    For small networks (i.e. 10 eNBs, 10 active subscribers), Magma can produce around 1 GB of logs per week. Most of these logs will go to /var/log/journal. To avoid problems with insufficient disk space, it is recommended to configure log rotation for systemd-journald.  For more information please visit [Ubuntu manuals](https://manpages.ubuntu.com/manpages/focal/man5/journald.conf.5.html).
```
Comment (gruyaume) [inline docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md]: This paragraph is not necessary in a how-to guide

File: docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+## Set up your management environment
+
+- Install [Juju 2.9](https://juju.is/docs/olm/installing-juju) on your management machine
+- Create and bootstrap a [manual cloud](https://juju.is/docs/olm/manual-setup)
+- [Add AGW host](https://juju.is/docs/olm/manual-setup#heading--add-machines-to-a-manual-cloud) to your manual cloud
```
Comment (gruyaume) [inline docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md]:
```suggestion
- [Add the Ubuntu machine](https://juju.is/docs/olm/manual-setup#heading--add-machines-to-a-manual-cloud) to your manual cloud
```

File: docs/readmes/lte/deploy_agw_using_juju.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
+---
+id: deploy_agw_using_juju
+title: Deploy AGW using Juju (Beta)
```
Comment (gruyaume) [inline docs/readmes/lte/deploy_agw_using_juju.md]:
```suggestion
title: Deploy AGW using Juju
```

File: docs/readmes/lte/deploy_agw_using_juju.md [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+hide_title: true
+---
+
+# Deploy Charmed Magma Access Gateway (Beta)
```
Comment (gruyaume) [inline docs/readmes/lte/deploy_agw_using_juju.md]:
```suggestion
# Deploy Charmed Magma Access Gateway
```

File: docs/docusaurus/i18n/en.json [hunk 9]
```diff
@@ -174,6 +174,9 @@
       "lte/debug_user_control_plane": {
         "title": "User Control Plane trace CLI"
       },
+      "lte/deploy_agw_using_juju": {
+        "title": "Deploy AGW using Juju (Beta)"
```
Comment (gruyaume) [inline docs/docusaurus/i18n/en.json]:
```suggestion
        "title": "Deploy AGW using Juju"
```

File: docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md (hunk 4 above)
Comment (Gmerold) [reply docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md]: In this case, Ubuntu is actually required. We can't run AGW on other OSes.

File: docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md (hunk 5 above)
Comment (Gmerold) [reply docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md]: Removed.

File: docs/docusaurus/versioned_docs/version-1.8.0/orc8r/deploy_using_juju.md (hunk 1 above)
Comment (Gmerold) [reply docs/docusaurus/versioned_docs/version-1.8.0/orc8r/deploy_using_juju.md]: True. Removed.

File: docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md (hunk 4 above)
Comment (gruyaume) [reply docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md]: The OS is listed in the bullet list below

**Review by gruyaume — APPROVED** [review]

File: docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md (hunk 4 above)
Comment (Gmerold) [reply docs/docusaurus/versioned_docs/version-1.8.0/lte/deploy_agw_using_juju.md]: True. Removed.

**Comment by lucasgonze** [comment]
@Gmerold can you fix the failing tests and rebase?

**Comment by Gmerold** [comment]
> @Gmerold can you fix the failing tests and rebase?

Hi @lucasgonze
Errors in the [Markdown lint check](https://github.com/magma/magma/actions/runs/4958875838/jobs/8872291749?pr=15186#logs) are not related to this PR and I think it would be better if they were handled in a separate PR.
As for the [Check for broken URLs in markdown files](https://github.com/magma/magma/actions/runs/4958875838/jobs/8872292224?pr=15186#logs), frankly I don't even see the actual error there... Would you be able to help on that?

**Review by jordanvrtanoski — APPROVED** [review]

**Pull request merged**